docker_run_worker_monitor.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh -e
+
+if [ ! -d "/data/workers" ] ; then
+  mkdir -p /data/workers
```

Comment:
for some reason this wouldn't work without the -p (I thought it was redundant as well), I think it has to do with a race condition with making the bound volume available? Not sure, I like the idea of sidestepping that all together

Reply:
you needed the `-p` because `/data` didn't exist. However, if you are going to run with `-p` there's no benefit to checking if the directory existed in the first place.

Reply:
now using `tmpfs` -- worked like a charm